docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -37,6 +38,80 @@
 | [v1.0.1](Data_Portal_Release_Notes.md#release-101) | May 18, 2016 |
 
 ---
+## Release 2.2.0
+
+* __GDC Product__: GDC Data Portal
+* __Release Date__:  June 26, 2024
+
+### New Features and Changes
+
+* __GDC 1.0 (Legacy Version)__:
+    * The legacy version (GDC 1.0) of the Data Portal has been officially retired.
+    * GDC 1.0 can no longer be reached.
+* __Annotations__:
+    * Annotations tables have been added to the __project, case, and file summary pages__. <!--PEAR-1853/1839/1855-->
+    * Links to the case summary page have been removed for annotations that concern a redaction of the case. <!--PEAR-1942-->
+* __Cohorts__ created from analysis tools now consistently consist of a specific list of cases that will remain unchanged after a data release. This includes cohorts created from the gene and mutation summary pages. <!--PEAR-1911-->
+* The files tables in the __Cart__ and the __Repository__ now allow searching for files based on the associated cases' submitter ID and UUID. <!--PEAR-1900-->
+* In the __case summary page__, values whose units are days, e.g. Days to Death or Days to Birth, are now displayed in years and days as appropriate for the user's convenience. <!--PEAR-1467-->
+* __Quick Search__ results and the headers of all __summary pages__ have been updated with new designs and icons <!--PEAR-1301-->
+* Word wrapping has been improved for __Quick Search__ results to avoid unexpected word breaks. <!--PEAR-1897-->
+* The Best Overall Response card in the Treatments category of the __Cohort Builder__ has been moved to a new position in the category. <!--PEAR-1988-->
+* The text referencing the deletion of custom sets in the __Manage Sets__ page has been updated. <!--PEAR-1939-->
+* The Access column has been added to the Source Files and Download Analyses Files tables in the __file summary page__. <!--PEAR-1872-->
+* Text within the downloaded histogram image has been updated for greater clarity in the __Clinical Data Analysis__ tool. <!--PEAR-1805--> 
+* Filter panels in the __Projects__, __Mutation Frequency__, and __Repository__ tools have been standardized and now consistently allow scrolling to occur independently of the tables on the right. <!--PEAR-1579-->
+
+### Bugs Fixed Since Last Release
+* __Section 508 Accessibility__:
+    * Aria labels have been added to the tables in Set Operations. <!--PEAR-1936-->
+    * The Venn diagrams in __Set Operations__ and __Cohort Comparison__ now have the appropriate alt text and roles <!--PEAR-1936-->
+    * The Venn diagram button in __Cohort Comparison__ has been updated with both an aria label and an informative label. <!--PEAR-1936-->
+    * The statistics table and its TSV in __Clinical Data Analysis__ now contain data for Q1 and Q3. <!--PEAR-1935-->
```

Comment:
should we add here that data for Q1 and Q3 was added only for the Box plot and the QQ plot?

Reply:
wouldn't hurt to clarify!

Reply:
adjusted!